test/e2e/infra/vagrant/provision.sh

```diff
@@ -1,10 +1,15 @@
 #!/usr/bin/env bash
 
 function usage() {
-    echo "Usage: provision.sh [--ip-family <v4|v6>] [-h|--help]"
+    echo "Usage: provision.sh [--ip-family <v4|v6>] [-l|--large] [-h|--help]
+    Provisions the Vagrant VMs.
+	  --ip-family <v4|v6>  Deploy IPv4 or IPv6 Kubernetes cluster.
+    --large              Deploy large vagrant VMs with 2 vCPUs and 4096MB memory.
```

Comment:
still an alignment issue?

Reply:
Yes fixed it properly and checked on github again.. editor was rendering it fine.